doc/tutorials/content/compiling_pcl_docker.rst

```diff
@@ -0,0 +1,154 @@
+.. _compiling_pcl_docker:
+
+Compiling PCL from source on Docker
+===================================
+
+This tutorial explains how to build the Point Cloud Library **from source** on docker.
```

Comment:
A short explanation what advantages (and disadvantages?) this has, and in which situations someone should choose this method, would be good. Maybe: does not install anything on your computer, no need to install dependencies, ...